src/graphic/minimap_renderer.cc

```diff
@@ -95,6 +95,9 @@ inline RGBColor calc_minimap_color(const Widelands::EditorGameBase& egbase,
 			Widelands::Coords starting_pos;
 			for (uint32_t p = 1; p <= map.get_nrplayers(); p++) {
 				starting_pos = map.get_starting_pos(p);
+				if (!static_cast<bool>(starting_pos)) {
```

Comment:
although explicit cast to bool is ok from our coderules, personally I like the check for nullptr better

Reply:
`starting_pos` is not a pointer so it cannot be `nullptr`. The `Coords::operator bool` is actually just a function that checks whether the coords is not a special "invalid" constants. We could refactor this into a function `is_valid()` or such instead of a `bool` conversion operator, but that's not related to this branch.

Reply:
ok I understand. and sure it is not related therefore i just commented and did not start a review